FRINXio/frinx-frontend
# Fix workflow manager bugs

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-478` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  | No

https://frinxhelpdesk.atlassian.net/browse/FD-478?atlOrigin=eyJpIjoiZWY3OGQ0ZDk2ZjQwNGVkYWIyMTZjZGE1Yzc0OGRjMTkiLCJwIjoiaiJ9

HOW TO TEST => 
1) Go to workflow manager > Definitions and try to create a new definition, insert a name, hit "Save changes" and now hit "Action" menu > Delete workflow. When user cant delete workflow because of no definition, a toast-notification appears

2) Go to workflow manager > Tasks. When tried to search in search bar, pagination showed multiple result pages even if there was no result per page or just a few per page. (Try searching "claim"). Fixed to work as usual

3) Go to workflow manager > Tasks. When deleted a task, the task stayed on the page until refresh and pagination shrinked to only 1 result page. Fixed so the deleted task is removed instantly and pagination works as it should
